renderer/src/common/mocks/fixtures/groups_name/delete.ts

```diff
@@ -0,0 +1,3 @@
+import type { DeleteApiV1BetaGroupsByNameResponse } from '@api/types.gen'
+
+export default '' satisfies DeleteApiV1BetaGroupsByNameResponse
```

Comment:
it is weird 🤔

Reply:
let me check in real life

Reply:
yup, this is 100% the actual response from BE. an empty string

Reply:
<img width="996" height="683" alt="Screenshot_select-area_20250916131016" src="https://github.com/user-attachments/assets/fb7ee738-91b8-410d-a5af-e001cdd7d08e" />